europe/README.EUROPE

```diff
@@ -294,3 +294,111 @@ Residuals at 130 further test points average 0.06m, maximum 0.16m.
+
+* ISN2004_ISN2016.gsb
+
+
+
+### Iceland: Deformation model: ISN93 -> ISN2016 
+
+*Source*: [IGN France](https://geodesie.ign.fr/index.php?page=grilles)   
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN93 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN93 3D geodetic (EPSG:4945) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN93_ISN2016.gsb
+* ISN93_ISN2016.gtx
+
+
+### Iceland: Deformation model: ISN2004 -> ISN2016 
+
+*Source*: [IGN France](https://geodesie.ign.fr/index.php?page=grilles)   
+*Format*: NTv2,GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Deformation model used in transformation between ISN2004 reference frame and ISN2016 reference frame
+Horizontal transformation is designed so that the ISN2016 points should fit within 1-2 cm accuracy, but in the area around the earthquake area in the South and North of Vatnajökull the accuracy is worse.
+Vertical transformation 
+Recommended to transform from ISN2004 3D geodetic (EPSG:5323) to ISN2016 3D geodetic (EPSG:8085)
+
+See the `ISL` init-file for examples of use.
+
+* ISN2004_ISN2016.gsb
+* ISN2004_ISN2016.gtx
+
+
+### Iceland: ISN2004 ellipsoidal height  -> ISH2004 vertical datum
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Geoid usually fits within 1-2cm, but in some areas it‘s slightly worse 3-5cm. The std for the fitted geoid is 1.7cm based on over 300 gnss levellin points
+
+Grid to convert from ISN2004 ellipsoidal height (EPSG:5323) to national vertical datum ISH2004 (EPSG:8089)
+The horizontal grid coordinates are referenced to ISN2004
+
+See the `ISL` init-file for examples of use.
+
+* icegeoid_2011.gtx 
+
+### Iceland: Velocity Model
+
+*Source*: [National Land Survey of Iceland](https://atlas.lmi.is/LmiData/index.php?id=626468364600)
+*Format*: CTable2 and GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+*Credit*: (c)2017-2019 Grid created by Guðmundur Valsson.
+
+Used in transformations between ISN2016 and global reference frame realisation ITRF14 with different epochs, 
```

Comment:
probably nitpicking, but ITRF14 didn't immediately resolve to ITRF2014 in my mind. Everywhere else in PROJ and in EPSG, the name ITRF2014 is used.

Reply:
Yes, ITRF2014 is the official name. I think confusion often arise because we also have IGS14 for GNSS processing, which is almost the same thing as ITRF2014.

Reply:
Thank you for pointing that out. I have edited the file regarding that.